api/.env.example

```diff
@@ -33,4 +33,10 @@ MAIL_SERVER=localhost
 LOCALSTACK_SERVER=http://localhost:4566
 
 QUEUE_URL=http://localhost:4566/000000000000/your-queue-name
-SQS_ENDPOINT=http://localhost:4566
\ No newline at end of file
+SQS_ENDPOINT=http://localhost:4566
+
+LIST_USERS_API_KEY=12345
```

Comment:
LIST_USERS_API_KEY and SLACK_CHANNEL_EMAIL have been in the app for a while but were missing from the example env file so added them here as well.